ckanext/dgu/theme/templates/issues/all_issues.html

```diff
@@ -1,54 +1,118 @@
+            <option value="least_recently_updated" {% if sort == "least_recently_updated" %}selected="selected"{% endif%}>Least Recently Updated</option>
+            <option value="most_commented" {% if sort == "most_commented" %}selected="selected"{% endif%}>Most Commented</option>
+            <option value="newest" {% if sort == "newest" %}selected="selected"{% endif%}>Newest</option>
+            <option value="oldest" {% if sort == "oldest" %}selected="selected"{% endif%}>Oldest</option>
+            <option value="recently_updated" {% if sort == "recently_updated" %}selected="selected"{% endif%}>Recently Update</option>
+        </select>
+        <button class="btn js-hide" type="submit">Go</button>
+      </div>
+    </form>
+    <div class="clearfix"></div>
+
+      <h2>
+        {{ ungettext('{number} issue found', '{number} issues found', pagination.total_count) .format(number=pagination.total_count) }}
+      </h2>
+    </div>
+    <div id="issue-page">
+      {% if issues %}
+        <div>
+          <table class="table">
+            <thead>
+              <tr>
+                <th></th>
+                <th>Title</th>
+                <th>Dataset</th>
+                <th>Reported</th>
+                <th>Status</th>
+                <th>Comments</th>
+              </tr>
+            </thead>
+            <tbody>
+            {% for issue in issues %}
+             {% set issue_url = h.url_for('issues_show',dataset_id=issue.dataset.name,issue_number=issue.number) %}
+              <tr class="{{'success' if issue.resolved else 'error'}}">
+                <td>
+                  <a href="{{ issue_url }}">
+                    <span class="list-group-item-number">#{{issue.id}}</span>
+                  </a>
+                </td>
+                <td>
+                  <a href="{{ issue_url }}">
+                    {{issue.title}}
+                  </a>
+                  {% if issue.resource %}
+                    <br/><br/>
+                    Resource: {{issue.resource.name or issue.resource.description}}
+                  {% endif %}
+                  {% if issue.visibility == 'hidden' %}
+                    <br/><br/>
+                    <div class="issue-comment-label">
+                      Reported as spam/abuse
+                    </div>
+                  {%endif%}
+                </td>
+                <td>{{issue.dataset.title}}</td>
+                <td><span class="timeago" title="{{h.render_datetime(issue.created, with_hours=True)}}">{{ h.time_ago_from_timestamp(issue.created) }}</span></td>
+                <td>{% if issue.resolved %}<span title="Closed by {{issue.resolver}} on {{issue.resolved}}">closed</span>{% else %}open{% endif %}</td>
+                <td>
+                  {% if issue.comment_count %}
+                    <i class="icon-comments"></i>
+                    {{ _('%s comments') % (issue.comment_count) }}
+                  {% endif %}
```

Comment:
As a sysadmin I want to be able to see the comments on the same page and moderate them. In which case, I think this page needs a bit of a rethink, since the use case is to moderate both new issues and comments that have come in recently.